SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -37,8 +37,9 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 	bool IsReady() const { return LocalVirtualWorkerId != SpatialConstants::INVALID_VIRTUAL_WORKER_ID; }
 
 	VirtualWorkerId GetLocalVirtualWorkerId() const { return LocalVirtualWorkerId; };
-	void SetLocalVirtualWorkerId(VirtualWorkerId LocalVirtualWorkerId);
+	virtual void SetLocalVirtualWorkerId(VirtualWorkerId LocalVirtualWorkerId);
 
+	// Deprecated: will be removed ASAP.
 	virtual TSet<VirtualWorkerId> GetVirtualWorkerIds() const PURE_VIRTUAL(UAbstractLBStrategy::GetVirtualWorkerIds, return {};)
```

Comment:
Can we remove this yet?

Reply:
Alas no - we have a NetTest project that's in a different repo that relies on this. As soon as this code goes in, I'll do a PR in that repo to switch to using GetMinimumRequiredWorkers and then I can remove this in the next PR.